Source/ACE.Entity/AceObjectPropertiesSpell.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using ACE.Common;
+using MySql.Data.MySqlClient;
+namespace ACE.Entity
+{
+    [DbTable("ace_object_properties_spell")]
```

Comment:
No problem.  I didn't realize the table had already existed.  Wanted to make sure we weren't making unnecessary DB changes.  This is fine.